codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddDocumentClientPlugin.java

```diff
@@ -123,7 +123,12 @@ private void writeAdditionalFiles(
                 writer.write("export * from './pagination';");
                 writer.write("export * from './$L';", DocumentClientUtils.CLIENT_NAME);
                 writer.write("export * from './$L';", DocumentClientUtils.CLIENT_FULL_NAME);
-                writer.write("export { NumberValueImpl as NumberValue } from \"@aws-sdk/util-dynamodb\";");
+                writer.write("");
+                writer.write("""
+                    export { NumberValueImpl as NumberValue } from "@aws-sdk/util-dynamodb";
+                    export { marshallOptions, unmarshallOptions } from "@aws-sdk/util-dynamodb";
+                    export { NativeAttributeValue, NativeAttributeBinary, NativeScalarAttributeValue } from "@aws-sdk/util-dynamodb";
+                    """);
```

Comment:
Just double checking, these additional exports are intentional?

Reply:
yes, they were manually added in the `.ts` file but now I'm adding it to codegen